paparazzi/libs/layoutlib/build.gradle

```diff
@@ -1,4 +1,5 @@
-apply plugin: 'org.ajoberstar.grgit'
+import org.ajoberstar.grgit.Grgit
```

Comment:
should we also remove the plugin and put `grgit-core` on the classpath instead? 
 https://search.maven.org/search?q=a:grgit-core

Reply:
Makes sense, looks a bit strange in code, but did it anyway, because the grgit-gradle classes are not used. c2cacaed